client/src/components/Overview/StyleSelector 2.jsx

```diff
@@ -0,0 +1,85 @@
+import React, {useState, useEffect} from 'react';
+
```

Comment:
Potentially a duplicate

Reply:
Dont know how that happened. Just deleted it.